src/test/java/uk/gov/justice/digital/delius/controller/secure/CustodyController_replaceNomsNumberTest.java

```diff
@@ -37,6 +39,7 @@
 
     @Before
     public void setup() {
+        RestAssuredMockMvc.config =  newConfig().encoderConfig(encoderConfig().defaultContentCharset("UTF-8"));
```

Comment:
Best solution I could find. It is connected to SpringBoot no longer defaulting charset which seems to only affect RestAssuredMockMVC (the RestAssured integration tests are fine, so it is only when RestAssuredMockMVC starts Spring we have the issue).